riot/riot.py

```diff
@@ -86,6 +86,10 @@ def version(self) -> str:
         version = output.decode().strip().split(" ")[1]
         return version
 
+    @functools.lru_cache()
```

Comment:
https://docs.python.org/3/library/functools.html#functools.cached_property :(

Reply:
I didn't notice before, this is 3.8 only :(